tests/basic/vcluster_test.go

```diff
@@ -1265,6 +1265,20 @@ var _ = Describe("{AutopilotMultipleFioOnManyVclusters}", func() {
 	})
 })
 
+var _ = Describe("{DeployMultipleKubevirtApps}", func() {
```

Comment:
This is a reminder to remove this before merging the PR if it is not an actual test case.

Reply:
time being we need it so keeping it